metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -287,26 +289,6 @@ def paginate_entity_api_results(self, entity_type, page_size=100):
 
             current_page += 1
 
-    @lru_cache(maxsize=None)
-    def get_platform_from_database_id(self, database_id):
```

Comment:
I don't see how we can get rid of this method- the purpose here was to stitch from superset into the upstream warehouse. Do we not need that logic anymore?